opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/PendingToken.java

```diff
@@ -34,11 +35,12 @@
     public final String str;
     public final int start;
     public final int end;
+
     /**
      * When tokenizers allow overlapping tokens, the following field is set to
      * {@code true} for tokens that should not increment the position attribute.
      */
-    public boolean nonpos;
+    boolean nonpos;
```

Comment:
It seems this is not used, even 3 years after it was introduced. I'd personally follow YAGNI and remove it.

Reply:
okay, removed.